tests/simplex/simplex_project_dg.output

```diff
@@ -35,3 +44,12 @@ DEAL::max error = 0.000842371
 DEAL::ratio = 7.61471
 DEAL::max error = 0.000106584
 DEAL::ratio = 7.90336
+DEAL::FE = FE_SimplexDGP<3>(3)
+DEAL::max error = 0.00566152
+DEAL::ratio = 176.631
+DEAL::max error = 0.000454735
+DEAL::ratio = 12.4501
+DEAL::max error = 3.01010e-05
+DEAL::ratio = 15.1069
+DEAL::max error = 1.90799e-06
+DEAL::ratio = 15.7764
```

Comment:
The serial CI complains that this number should be `15.7763`. I am not sure if one should increase the number of digits (because the number happens to be right in the middle and changes in the last few digits are present), or if we should reduce the number of digits by 1 because we can only expect approximate results. @drwells could you please have a look, I think apart from this everything should work.

Reply:
I made the test easier (L2 instead of Linfty) and also trimmed the output.